starter-code/database.js

```diff
@@ -72,8 +73,9 @@ class Database {
         // Implement the query to delete a user
         // firstName is the name of user that we want to delete
         // remeber once it's finish to comment callback('Error deleting user');
+        database.collection('users').deleteOne(firstName, callback);
```

Comment:
I'm not sure if you understand that, what you have written here is equivalent to:
```
database.collection('users').deleteOne('firstName' : firstName, callback);
```
Here, of course, works, but not in line 123 (see that comment).